lib/services/plugins/nativescript-project-plugins-service.ts

```diff
@@ -175,7 +186,15 @@ export class NativeScriptProjectPluginsService extends NpmPluginsServiceBase imp
 
 	public getPluginBasicInformation(pluginName: string): IFuture<IBasicPluginInformation> {
 		return ((): IBasicPluginInformation => {
-			let [name, version] = pluginName.split("@");
+			let name: string, version: string;
+			let scopedDependencyRegex = /^(@.+?)(?:@(.+?))?$/;
```

Comment:
The combination of `+?` is equivalent to `*`, not a merge-stopper but the two occurrences can be replaced and therefore shortened